### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with Keyboard Navigation in Login/Signup Screen

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4764]
# Intro
This is an accessibility problem with keyboard navigation I found in the Login/Signup screens.

## Issue
Keyboard navigation works to some extent, but cannot be used to switch between login and signup. ***This is a violation of WCAG Success Criterion 2.1.1 (Level A) - keyboard***. [See](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

## Description
***Success Criterion 2.1.1 (Level A) - keyboard*** requires that all functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

## Impact
The inability to fully navigate with a keyboard would affect users with visual impairments, screen readers, or alternative forms of navigation (other than a mouse). They would be unable to use our platform.

## Fix
Using a `tab-index` in the navigation tab for `Log in` and `Sign up`. I will make a PR for a fix shortly.
